tests/test_spelling.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""Testing the spelling of numbers
```

Comment:
`test_number_spelling.py` and this module seems do be duplicated (or at least very similar). Can you clean up the PR by deleting unused code?